packages/forma-36-react-components/CHANGELOG.md

```diff
@@ -3,77 +3,6 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
-## [3.6.6](https://github.com/contentful/forma-36/compare/@contentful/forma-36-react-components@3.6.5...@contentful/forma-36-react-components@3.6.6) (2019-05-07)
```

Comment:
rebase or checkout changelog before merge

Reply:
I created two branches with changes and requested your review. Those were my outstanding changes for copy. Thanks for your patience.